### ISSUE TITLE
 Failed to build gem native extension.

### ISSUE THREAD

**Issue by yoykiee** [oneclick/rubyinstaller#188]
Hi! Im trying setup rails on Windows 7.

I install ruby x64 from rubyinstaller.org

C:\DevKit>ruby -v
ruby 2.0.0p247 (2013-06-27) [x64-mingw32]

After, I install Ruby 2.0.0 x64 (64bits): mingw64-64-4.7.2
"ruby dk.rb init" doesn't find  installed ruby. 
After, I add mannualy ruby directory.
But,

C:\DevKit>ruby dk.rb install
[INFO] Skipping existing gem override for 'C:/Ruby200-x64'
[WARN] Skipping existing DevKit helper library for 'C:/Ruby200-x64'

I searched all the internets, help, please...

PS: Im trying with mingw64-32-4.7.2, I have same errors.

**Comment by nuriasakura** [comment]
I have the same exact problem. I can understand @yoykiee ...

I Installed ruby, rubygems and rubydev kit.
On the ruby website it says that I downloaded the 64x from ruby and from the dev kit.

The versions are:
C:\>ruby -v
ruby 2.5.0p0 (2017-12-25 revision 61468) [x64-mingw32]  (Note: I really downloaded the 64x version of the DevKit, however it appears as mingw32 on the terminal)
C:\> gem -v
2.7.6

Then I did the ruby dk.rb init; I manually added ruby's location (although it was already there)

and When I do the ruby dk.rb install it goes : 

C:\Users\nuria\RubyDevKit> ruby dk.rb install
[INFO] Skipping existing gem override for 'C:/Ruby25-x64'
[WARN] Skipping existing DevKit helper library for 'C:/Ruby25-x64'



I want to install jekyll. And It always ends up like: 

C:\> gem install jekyll
Building native extensions. This could take a while...
ERROR:  Error installing jekyll:
        ERROR: Failed to build gem native extension.





In other words, same situation.
PS: English is not my first language either but i do understand quite fairly.

**Comment by nuriasakura** [comment]
I notice that this is closed. I however I have the same exact problem.. and since there was no solution... Please reopen this issue ;-;

@luislavena  @sroller 

Also @yoykiee did you find the solution?

**Comment by larskanis** [comment]
@nuriasakura You're using RubyInstaller-2.5. It is not compatible to the old devkit.

Instead you should run `ridk install` in order to download and install MSYS2. The devkit registration per `ruby dk.rb init` is no longer necessary.

"x64-mingw32" means the true 64-bit version - the "32" in the name is for historical reasons.

**Comment by larskanis** [comment]
@nuriasakura If it still doesn't work for you, please open a ticket [at RubyInstaller2](https://github.com/oneclick/rubyinstaller2).

**Comment by nuriasakura** [comment]
@larskanis  thank you very much! It worked! 
ridk install and 3rd option, and it worked. thanks!!!
